docs/src/routes/docs/[...4]wallets/injected.md

```diff
@@ -268,6 +268,7 @@ const injected = injectedModule({
 ### Injected Wallets Supported Natively
 
 - Metamask - _Desktop & Mobile_
+- Safeheron - _Desktop_
```

Comment:
```suggestion
```
Please add to the bottom of the list, we add wallets here in the order in which they have added web3-onboard support